stratum/lib/security/auth_policy_checker.cc

```diff
@@ -59,6 +67,7 @@ void ReadProtoIfValidFileExists(const std::string& path,
   }
 }
 
+#ifdef __linux__
```

Comment:
Platform `ifdefs` in implementation code are not great. On the other hand, this class is mostly a stub and those functions are unused.